<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="Academy.TestComponents.Listeners"/>
    <listener class-name="Academy.TestComponents.TestRetryAnalyzerListener"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="ErrorValidations(failed)">
    <groups>
      <run>
        <include name="ErrorHandling"/>
      </run>
    </groups>
    <classes>
      <class name="Academy.Tests.ErrorValidationsTCPO">
        <methods>
          <include name="tearDown"/>
          <include name="loginWrongUsernameDisplayedTestCaseTransformer"/>
          <include name="loginWrongUsernameDisplayedTestCase"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- Academy.Tests.ErrorValidationsTCPO -->
    </classes>
  </test> <!-- ErrorValidations(failed) -->
</suite> <!-- Failed suite [Suite] -->
